index.html

```diff
@@ -57,8 +57,15 @@ <h3 class="main-subheading">
         assets at your fingertips.
```

Comment:
All block content cannot consist only of headings. A heading should title some content. In this block, level 2 and 3 headings do not title anything and therefore are not headings.

Try to create create a table of contents from headings as in a book, where h1 is the book title, h2 - chapters, h3 - sections within those chapters, h4 - subsections. If a heading does not title any text or section, it is not a heading.

Or use https://yoksel.github.io/html-tree/en/ for visualisation of headings tree.